documentation/Functions.md

```diff
@@ -159,9 +159,15 @@ installCobiGen()
 ***
 
 ### cobiGenJava <a name="cobiGenJava"></a>
+This function is used for code generation inside Java file i.e. Entity class in Java.
 #### parameter
-1. The path to the java file you want to generate code for: string
-2. The numbers that represent the templates that CobiGen uses to generate code: int array
+You need two parameters for this function.
+
+Example: cobiGenJava(1st-parameter, 2nd-parameter)
+
+1st-parameter: The path to the java file you want to generate code for. It is of type string.
```

Comment:
Java with leading capital letter

Reply:
@GuentherJulian we could also add relative to working directory to align it with the other functions. The implementation has to be changed. Currently, it´s relative to the main directory from the devonfw-ide installation.